server/service/expected.go

```diff
@@ -10,10 +10,10 @@ import (
 
 // Service is 'service/service.go'
 type Service interface {
-	EnclaveInfo() *sgx.EnclaveInfo
-	OracleAcc() *panacea.OracleAccount
-	OraclePrivKey() *btcec.PrivateKey
-	Config() *config.Config
-	QueryClient() panacea.QueryClient
-	IPFS() *ipfs.IPFS
+	GetEnclaveInfo() *sgx.EnclaveInfo
+	GetOracleAcc() *panacea.OracleAccount
+	GetOraclePrivKey() *btcec.PrivateKey
+	GetConfig() *config.Config
+	GetQueryClient() panacea.QueryClient
+	GetIPFS() ipfs.IPFS
 }
```

Comment:
This interface is duplicated with the `service.Service` interface. Do we need this interface as well?

Reply:
Yes. The reason I used this was because of the circular error.
For now, I'll check again to see if I've changed the structure and there's no problem.